Dockerfile

```diff
@@ -59,6 +59,7 @@ DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-unauthenticated \
     rocm-opencl \
     rocm-opencl-dev \
     rocblas \
+    rpm \
```

Comment:
why do we need it?

Reply:
@shurale-nkn  it is to get rid of a warning message at `cmake` stage, mostly cosmetic I agree

Reply:
@shurale-nkn  I am okay with removing it for now (see no harm having it as is too), but we need this PR or #1503 in `develop` branch or that branch is blocked by CI base docker update :)
We need either of these two as quickly as possible